explorer/src/lib/components/navbar/Navbar.svelte

```diff
@@ -60,7 +60,9 @@
     showSearchNotification = false;
   });
 
-  $: ({ networks } = $appStore);
+  $: ({ darkMode, networks } = $appStore);
+
+  const isDarkMode = darkMode;
```

Comment:
You should be able to directly bind to the `darkMode` from the appStore, I don't think this constant is needed.